src/Uno.UWP/UI/Colors.cs

```diff
@@ -71,45 +71,62 @@ public static Color Parse(string colorCode)
 		public static Color FromARGB(string colorCode)
 		{
 			byte a, r, b, g;
+			int offset;
+			int len;
 
-			colorCode = colorCode.TrimStart(new char[] { '#' });
+			if (colorCode is null)
+			{
+				len = 0;
+				offset = 0;
+			}
+			else
+			{
+				len = colorCode.Length;
+				// skip a starting `#` if present 
+				offset = (len > 0 && colorCode[0] == '#' ? 1 : 0);
+				len -= offset;
+			}
 
-			if (colorCode.Length == 3)
+			// deal with optional alpha value
+			if (len == 4)
 			{
-				a = 0xFF;
-				r = Convert.ToByte(new String(colorCode[0], 2), 16);
-				g = Convert.ToByte(new String(colorCode[1], 2), 16);
-				b = Convert.ToByte(new String(colorCode[2], 2), 16);
+				a = Convert.ToByte(colorCode[offset++]);
```

Comment:
This will produce the ASCII equivalent of the character. I don't think this is the intended behavior.

Reply:
nope, it was not :)